examples/aws/main.tf

```diff
@@ -1,7 +1,29 @@
 provider "gardener" {
   profile            = "<my-gardener-project>"
   aws_secret_binding = "<my-gardener-aws-secret>"
-  kube_path          = "<my-gardener-service-account-kubeconfig>"
+  kube_path          = "${file("<my-gardener-service-account-kubeconfig>")}"

```

Comment:
Shouldn't this property be called `kube_file`? Same for the other .tf files

Reply:
fixed it